src/Symfony/Component/Intl/Countries.php

```diff
@@ -52,6 +52,20 @@ public static function getAlpha3Codes(): array
         return self::readEntry(['Alpha2ToAlpha3'], 'meta');
     }
 
+    /**
+     * Returns all available numeric country codes (3 digits).
+     *
+     * Countries are returned as ISO 3166 numeric three-digit country codes.
+     *
+     * This list only contains "officially assigned ISO 3166-1 numeric" country codes.
+     *
+     * @return string[]
```

Comment:
this should be documented as `@return array<string, string>` to be more precise about the type of keys being returned, as this returns a mapping with alpha2 codes as keys (this should probably also document what the keys are)